pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/RealtimeNoDictionaryTimeColumnClusterIntegrationTest.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.integration.tests;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.apache.commons.io.FileUtils;
+import org.apache.pinot.util.TestUtils;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+
+/**
+ * Extends RealtimeClusterIntegrationTest and configure time column to be a non-dictionary column.
+ */
+public class RealtimeNoDictionaryTimeColumnClusterIntegrationTest extends RealtimeClusterIntegrationTest {
```

Comment:
Will this be run as a separate integration test (as in setup/teardown the cluster)? If so, perhaps it can be merged into `RealtimeClusterIntegrationTest` by choosing some of the columns as `noDictionary`?

Reply:
Yes, wanna check if time column could be set as dictionary or no-dictionary. It's a bit hard to merge it as only one time column is used during segment creation for metadata/segment name etc.

Reply:
One way to merge these 2 tests is by using a random boolean to decide the NoDictionaryColumns

Reply:
Then we may not catch a potential issue in the corresponding PR?

Reply:
is the idea to merge more about saving the time it takes to run the build?

Reply:
Yes. Adding each new integration test will make the testing time roughly 3 minutes longer.

Reply:
got it, added a random condition here.